dev/archery/archery/bot.py

```diff
@@ -291,11 +291,9 @@ def handle_issue_comment(self, command, payload):
                          comment=comment)
         except Exception as e:
             logger.exception(e)
-            url = "{server}/{repo}/actions/runs/{run_id}".format(
-                server=os.environ["GITHUB_SERVER_URL"],
-                repo=os.environ["GITHUB_REPOSITORY"],
-                run_id=os.environ["GITHUB_RUN_ID"],
-            )
+            url = (f"{os.environ['GITHUB_SERVER_URL']}/"
```

Comment:
I actually prefer the old version here, I think is is more readable with the placeholders.

Reply:
Sure, resolved in [97070b2](https://github.com/apache/arrow/pull/45629/commits/97070b285f695bd65fe955fe866338505f7962cc)